tests/sys/zuse/ethereum/encoding.hoon

```diff
@@ -42,3 +42,4 @@
             "48656c6c6f2c20776f726c642100000000000000000000000000000000000000"
         ==
 --
+[l+~ l+[l+~ ~] l+[l+~ l+[l+~ ~] ~]]
```

Comment:
Spurious change to trigger a sync?

Reply:
Wh- what is this? Looks like some test noun for RLP encoding, but why in this file? And why did it show up now? How did I not see this in the staged diff?

![mystery-smallest](https://user-images.githubusercontent.com/3829764/50146170-782b0e80-02b3-11e9-9f46-478318535cdc.png)